2016/index.html

```diff
@@ -1,42 +1,41 @@
 <!doctype html>
 <html lang="en">
-
-<head>
-
-  <meta charset="utf-8">
-
-  <!-- Content metadata -->
-  <title>Revolution UC</title>
-  <meta name="description" content="RevolutionUC - Student Hackathon held at the University of Cincinnati">
-  <meta name="author" content="Caleb Goodreau (https://github.com/CalebGoodreau and Michael Zalla (https://github.com/MichaelZalla)">
-  <!-- Behavioral metadata -->
-  <meta http-equiv="content-type" content="text/html;charset=utf-8">
-  <meta http-equiv="X-UI-Compatible" content="IE=edge,chrome=1">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0,user-scalable=no">
-
-  <!-- Favicons -->
-  <!-- [if IE]><link rel="shortcut icon" href="favicon.ico?v=0.0.1" /><![endif] -->
-  <link rel="icon" type="image/png" href="/favicon-32.png?v2" sizes="32x32" />
-  <link rel="icon" type="image/png" href="/favicon-16.png?v2" sizes="16x16" />
-  <link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png">
-
-  <!-- facebook meta -->
-  <meta property="og:title" content="RevolutionUC">
-  <meta property="og:description" content="Student Hackathon held at the University of Cincinnati October 8th and 9th, 2016">
-  <meta property="og:image" content="https://revolutionuc.com/images/logos/social-card.jpg">
-  <meta property="og:url" content="https://revolutionuc.com/">
-
-  <!-- twitter meta -->
-  <meta name="twitter:title" content="RevolutionUC">
-  <meta name="twitter:description" content="Student Hackathon held at the University of Cincinnati October 8th and 9th, 2016">
-  <meta name="twitter:image" content="https://revolutionuc.com/images/logos/social-card.jpg">
-  <meta name="twitter:card" content="summary_large_image">
-  <meta name="twitter:site" content="@revolution_uc">
-
-  <!-- Stylesheets -->
-  <link rel="stylesheet" href="/css/startup-sass.min.css?v=0.0.1">
-  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Ubuntu:400,700">
-</head>
+  <base href="/2016/">
+  <head>
+    <meta charset="utf-8">
+
+    <!-- Content metadata -->
```

Comment:
Add a `<meta name="robots" content="noindex, nofollow">` so search engines don't look at the old site